locale/es.po

```diff
@@ -716,121 +716,121 @@ msgstr "Todos los timers están siendo utilizados"
 
 #: ports/nrf/common-hal/busio/SPI.c:170
 msgid "Baud rate too high for this SPI peripheral"
-msgstr ""
+msgstr "Baud rate demasiado alto para este periférico SPI"
```

Comment:
Can circuit python deal correctly with accents?
Checking just in case, as these are "character modifiers" (the second character modifies the previous), and even if correctly encoded in UTF-8 some terminals cold also struggle to display them without additional configuration.
I think UTF-8 itself has "precomposed" characters, which are like a single unicode characters that could contain the vowel with an accent included, rather than having two characters (vowel + accent). It possible that might be preferable?

Perhaps this is a question for more for @dhalbert or @tannewt?

Reply:
Accents are fine. Please don't avoid them, yes, use the standard UTF-8 characters you'd use normally.

Reply:
Great, thanks @dhalbert!

Reply:
I used accents were needed, i will check if there are any missing accents.